projects/styles/basic/main.less

```diff
@@ -1,22 +1,37 @@
-html {
+html[data-tui-theme] {
```

Comment:
I don't think we should add this change here, it is a conscious decision to use our global styles, makes no sense to make them disableable like that.

Reply:
fixed